app/models/concerns/notable.rb

```diff
@@ -0,0 +1,15 @@
+module Notable
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :concrete_notes,
```

Comment:
Is this intended to be temporary (and will be called "notes" once we get rid of the existing column?). I'm wondering whether "`directly_attached_notes`" would be clearer, but if this is only a temporary, internal thing then doesn't matter much.

Reply:
Yeah we should rename to `notes`. Thinking about it a bit more maybe we should rename the current column to `notes_old` or something and rename this now as this would save waiting for a release before renaming.

Reply:
Would that prevent the existing notes from displaying? We'd have to update all calls (e.g in views to `public_body.notes_old`) and then change it back?

Reply:
Lets ignore this for now and get this PR reviewed and merged. We can deal with this in https://github.com/mysociety/alaveteli/issues/7216